ui/src/components/flows/ValidationError.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <span ref="rootContainer">
         <!-- Valid -->
-        <el-button v-if="!errors && !warnings &&!infos" v-bind="$attrs" :link="link" :size="size" type="default" class="success square">
+        <el-button v-if="!errors && !warnings &&!infos" v-bind="$attrs" :link="link" :size="size" type="default" class="success square" :disabled="isDisabled">
```

Comment:
Why not just add `disabled` attribute here and that's it? This button is only shown when it's valid, and that's what we need. So, can you revert other changes and from this file and only add `disabled` prop here and we're good to go?